tests/console/libssh.pm

```diff
@@ -84,12 +85,12 @@ EOT
     }
     #Build an custom image with openssh curl libvirt-client sshpass
     assert_script_run("mkdir /tmp/build && cd /tmp/build");
-    assert_script_run("cat > /tmp/build/Dockerfile <<'END'\n$dockerfile\nEND\n( exit \$?)");
+    type_string("cat > /tmp/build/Dockerfile <<'END'\n$dockerfile\nEND\n( exit \$?)\n\n");
     assert_script_run(qq(docker build -t libssh_image --build-arg tag="$tag" --build-arg pkgs="$pkgs" .), timeout => 600);
```

Comment:
Maybe for a follow up, to download the dockerfile instead?

Reply:
I personally like this way, you can directly see what is done, how does the Dockerfile looks like.
Straight as it gets, IMO can't be better :grin: